test/e2e/manifests/mixins/cluster-workflow-template-rbac.yaml

```diff
@@ -14,6 +14,18 @@ rules:
       - watch
       - create
       - delete
+  - apiGroups:
+      - argoproj.io
+    resources:
+      - workflows 
+      - workflows/finalizers
+    verbs:
+      - get
+      - list
+      - watch
+      - update
+      - create
+      - delete
```

Comment:
Add permissions to make the server can get workflows by the label selector.

Reply:
Does this mean that the server couldn't list workflows before? It seems like this role is already granted in `argo-server-role`. In any case, this role is specifically for cluster workflow templates, so this should most likely be removed

Reply:
I've missed that `argo-server-role` already has it. Thank you for detail.

Reply:
@simster7 I've stuck in the failure of `e2e-test`. When I tried `e2e-test` it failed by the permission issue like the following:

```
../../dist/argo -n argo terminate --selector argo-e2e=true
exit status 1time="2020-10-02T22:19:27+09:00" level=fatal msg="rpc error: code = PermissionDenied desc = workflows.argoproj.io is forbidden: User \"system:serviceaccount:argo:argo-server\" cannot list resource \"workflows\" in API group \"argoproj.io\" at the cluster scope"

--- FAIL: TestCLIWithServerSuite (7.81s)
    --- FAIL: TestCLIWithServerSuite/TestRetryByLabelSelector (7.77s)
        cli_test.go:693:
            	Error Trace:	cli_test.go:693
            	            				when.go:280
            	            				when.go:289
            	            				cli_test.go:692
            	Error:      	Received unexpected error:
            	            	exit status 1
            	Test:       	TestCLIWithServerSuite/TestRetryByLabelSelector
```

I think it should be success because the `e2e-test` is using the token of the `argo-server` SA which has the `list` permission of workflow ([code](https://github.com/argoproj/argo/blob/master/test/e2e/fixtures/e2e_suite.go#L141)). Could you let me know how to fix it?